lib/bots/versionbot.ts

```diff
@@ -856,7 +856,7 @@ export class VersionBot extends ProcBot {
 
 		// If there's an unlabeling event, but it wasn't the IgnoreLabel being removed, then we
```

Comment:
Should this comment have been updated to match the code? Otherwise a future reader won't know if the comment is incorrect or the code is incorrect! :-/